extensions/wikia/UserActivity/UserActivityController.class.php

```diff
@@ -58,16 +54,16 @@ public function index() {
 	protected function formatItems( $items ) {
 		$flattened = [];
 		foreach ( $items as $sortKey => $contribItem ) {
-			$lastEditTS = wfTimestamp( TS_MW, $contribItem[ 'last_edit' ] );
+			$lastEditTS = wfTimestamp( TS_MW, $contribItem[ 'lastedit' ] );
+			unset($contribItem['lastedit']);
+
 			$localizedDate = \F::app()->wg->Lang->timeanddate( $lastEditTS, $localTZ = true );
 			$contribItem['lastEdit'] = $localizedDate;
-			unset($contribItem['last_edit']);
 
-			$editCount = $contribItem['editcount'];
+			$editCount = $contribItem['edits'];
+			unset($contribItem['edits']);
```

Comment:
To be clearer, I was referring to the [constants here](https://github.com/Wikia/app/pull/9357/files#diff-b28e18ad904e400ccc6038595de3ce0bR8). It would be neat if they could be in a common place an accessed in this class too, but not a big deal.